File: 3131.找出与数组相加的整数-i.java
/*
 * @lc app=leetcode.cn id=3131 lang=java
 *
 * [3131] 找出与数组相加的整数 I
 */

// @lc code=start
class Solution {
    public int addedInteger(int[] nums1, int[] nums2) {
        
        return Arrays.stream(nums2).min().getAsInt() - Arrays.stream(nums1).min().getAsInt();
    }
}
// @lc code=end

File: 3131.找出与数组相加的整数-i.js
/*
 * @lc app=leetcode.cn id=3131 lang=javascript
 *
 * [3131] 找出与数组相加的整数 I
 */

// @lc code=start
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number}
 */
var addedInteger = function(nums1, nums2) {
    
    return Math.max(...nums2) - Math.max(...nums1);
};
// @lc code=end

